### DISCUSSION TITLE
Search All Models API

### DISCUSSION THREAD

**Discussion by renatoalmeidaoliveira** [netbox-community/netbox#8144] [Q&A] [answered]
In the Web UI there're search field for Query all models, there're any API call to do that, or I have to implement a call for each model with `?q=<query>` ?

**Comment by jeremystretch** [comment, marked as answer]
There's no API endpoint to query that will return various models I'm afraid. While this might be feasible to implement, it would likely be difficult to consume.

**Reply by renatoalmeidaoliveira** [reply]
Thanks!
